config/24.1/make.conf

```diff
@@ -123,8 +126,4 @@ CADDY_CUSTOM_PLUGINS=		github.com/caddyserver/ntlm-transport \
 				github.com/caddy-dns/hexonet \
 				github.com/caddy-dns/mailinabox
 
-CADDY_CUSTOM_PLUGINS_BROKEN=	github.com/caddy-dns/netcup \
-				github.com/caddy-dns/desec \
-				github.com/caddy-dns/porkbun \
-				github.com/caddy-dns/netlify \
-				github.com/caddy-dns/namesilo
+CADDY_CUSTOM_PLUGINS_BROKEN=	github.com/caddy-dns/vultr
```

Comment:
do you still need this? the _BROKEN thing I just put here for further reference but it has no effect and when you ship without vultr then we can drop it

Reply:
Removed mention of it, when it's fixed in the future I will evaluate the build including it again.